deploy/manifests/prod/us-east-2/cluster/cert-manager/issuer.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt
+spec:
+  acme:
+    server: https://acme-v02.api.letsencrypt.org/directory
+    email: bedrock@protocol.ai
+    privateKeySecretRef:
+      name: letsencrypt
+    solvers:
+      - dns01:
+          route53:
+            region: us-east-2
+        selector:
+          dnsZones:
+            - prod.cid.contact
```

Comment:
if another entity wanted to deploy their own deployment of store the index, they'd need to change these variables. is there a way to template these specific names/email addresses out?

Reply:
Context: Manifests built using `Kustomization` overlays patch some base layer to add/edit the things they need to change. This means there is no templating as such. One would simply define a patch and use strategic merge to override values.

In the case of storetheindex, the base kustomization does not define `Ingress` object at all. Because we don't know how other people may choose to expose their indexer. This is why the ingress object is defined in `prod` and `dev` overlays explicitly. The `ingress` object is also somewhat Cloud-provider specific (you can see route53 conifig in there).

Another reason for not including it in base is that in kustomization overlays one can't remove manifests. So things like this are left out so that whoever is using the kustomization can configure the right abstraction for themselves.